mbuild/formats/par_writer.py

```diff
@@ -0,0 +1,122 @@
+import mbuild as mb
+import parmed as pmd
+import warnings
+from foyer import Forcefield
+
+__all__ = ['write_par']
+
+
+def write_par(structure, filename):
+    """ Write CHARMM Par file given a parametrized structure """
```

Comment:
This is not urgent, but we should add more thorough documentation like the rest of our modules so that we don't have to worry about it at the next docathon. And we should add a link to the par/prm format page (https://www.ks.uiuc.edu/Training/Tutorials/namd/namd-tutorial-unix-html/node25.html) in the comments for future reference since the rest of the code is based on this.